theano/tensor/basic.py

```diff
@@ -5066,9 +5013,6 @@ def infer_shape(self, node, shapes):
             return [xshp[:-1] + yshp[-1:]]
         raise NotImplementedError()
 
-    def __str__(self):
-        return "dot"
```

Comment:
The change makes it return `"Dot"` instead of `"dot"`, which messes up with other tests.
Restoring the `__str__` method here seems easier than fixing the other tests.